source/_docs/domains.md

```diff
@@ -0,0 +1,182 @@
+categories: []
+---
+## Platform domains
+Every site on Pantheon is given multiple environments (Dev, Test, and Live), each with their own unique platform domains which are bound to isolated resources, and are uniquely addressable on the internet via Pantheon's own DNS. [Multidev](/docs/multidev/) environments get the same treatment. Platform domains are configured as subdomains, and are generated by prefixing the environment to the site name:
+
+- dev-site-name.pantheonsite.io
+- test-site-name.pantheonsite.io
+- live-site-name.pantheonsite.io
+- multidev-env-site-name.pantheonsite.io
+
+### robots.txt
+Pantheon serves a default robots.txt that disallows crawlers on platform domains (`/*.pantheonsite.io`, `/*.pantheon.io`, `/*.gotpantheon.com`, and `/*.sites.my-agency.com`). Crawlers are allowed on the Live environment to requests served with a custom domain (e.g., `www.example.com`). If you attempt to access your live environment with a platform-issued domain, even if you have a domain associated with the environment, the default robots.txt will be served.
+
+Pantheon does not allow crawlers on Dev, Test, or Multidev environments. Adding a custom domain to an environment other than Live will not permit crawlers to that environment.
+
+## Custom domains
+Custom domains are domains registered at a third-party provider, for which you control DNS configurations. Connect a custom domain to the Site Dashboard and point DNS at Pantheon to trigger the [automated process of provisioning HTTPS](/docs/https/).
+
+A paid plan is required to connect custom domains to an environment in the Pantheon Site Dashboard, up to the following limits:
+
+<table class="table table-condensed table-bordered">
+  <thead class="thead-inverse">
+    <tr>
+      <th scope="row" class="thead-inverse"></th>
+      <th>Personal</th>
+      <th>Professional</th>
+      <th>Business</th>
+      <th>Elite</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row" class="thead-inverse">Custom Domain Limit (per site)</th>
+      <td>5</td>
+      <td>25</td>
+      <td>100</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <th scope="row" class="thead-inverse">Free and managed HTTPS</th>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+    </tr>
+  </tbody>
+</table>
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Add all domains you want to resolve to Pantheon within the Site Dashboard for each respective environment, as described in [Launch Essentials](/docs/guides/launch/). Automatic resolution of domains and wildcards are not supported.</p></div>
+
+### Primary domain
+A primary domain refers to a single custom domain used to serve responses, regardless of originating URL of a request (e.g., `https://www.example.com`). Configure the primary domain as part of the going live process described in [Launch Essentials: Redirect to a Primary Domain](/docs/guides/launch/redirects/).
```

Comment:
Pantheon uses the term "primary domain" to refer to a domain to redirect all traffic. So for example, if you choose www.myawesomesite.com as your primary domain, then if a visitor types in myawesomesite.com into their browser (or any other domain you have connected to your site), they will end up www.myawesomesite.com. Configure the domain....